dashboard/src/pages/TreeDetails/Tabs/Tests/IndividualTestsTable.tsx

```diff
@@ -79,12 +80,48 @@ const columns: ColumnDef<TIndividualTest>[] = [
     cell: ({ row }): string =>
       row.getValue('duration') ? row.getValue('duration') : '-',
   },
+  {
+    id: 'hardware',
+    accessorKey: 'hardware',
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'treeDetails.hardware',
+        intlDefaultMessage: 'Hardware',
+      }),
+    cell: ({ row }): JSX.Element => (
+      <TooltipHardware hardwares={row.getValue('hardware')} />
+    ),
+  },
   {
     id: 'chevron',
     cell: (): JSX.Element => <ChevronRightAnimate />,
   },
 ];
 
+const TooltipHardware = ({
+  hardwares,
+}: {
+  hardwares: string[] | undefined;
+}): JSX.Element => {
+  return (
+    <Tooltip>
+      <TooltipTrigger>{hardwares ? hardwares[0] : '-'}</TooltipTrigger>
```

Comment:
you can use `sanitizeTableValue(hardwares[0], false)` just to have the empty value `'-'` being set in one place